app/requirements.txt

```diff
@@ -0,0 +1,5 @@
+django-cors-headers==3.11.0
+djangorestframework==3.13.1
+psycopg2-binary==2.9.2
+boto3==1.21.8
+fleep
```

Comment:
version not specified, very bad practice

Reply:
done